--- a/persist/log.go
+++ b/persist/log.go
@@ -6,6 +6,7 @@
 	"log"
 	"os"
 	"sync"
+	"path/filepath"
 
 	"github.com/NebulousLabs/Sia/build"
 )
@@ -119,7 +120,7 @@
 // NewFileLogger returns a logger that logs to logFilename. The file is opened
 // in append mode, and created if it does not exist.
 func NewFileLogger(logFilename string) (*Logger, error) {
-	logFile, err := os.OpenFile(logFilename, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0660)
+	logFile, err := os.OpenFile(filepath.Join("/var/log/sia", filepath.Base(logFilename)), os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0660)
 	if err != nil {
 		return nil, err
 	}
