From: Bjorn Dolk <debian@bjorndolk.com>
Date: Thu, 18 May 2017 21:50:33 +0200
Subject: inconshreveable-muxado

---
 modules/gateway/peers.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/gateway/peers.go b/modules/gateway/peers.go
index 24680e7..71a4789 100644
--- a/modules/gateway/peers.go
+++ b/modules/gateway/peers.go
@@ -10,7 +10,7 @@ import (
 	"github.com/NebulousLabs/Sia/encoding"
 	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/fastrand"
-	"github.com/NebulousLabs/muxado"
+	"github.com/inconshreveable/muxado"
 )
 
 var (
