From: Bjorn Dolk <debian@bjorndolk.com>
Date: Thu, 18 May 2017 22:00:22 +0200
Subject: dchest-blake2b

---
 crypto/hash.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/hash.go b/crypto/hash.go
index 4ac3452..1e60ae0 100644
--- a/crypto/hash.go
+++ b/crypto/hash.go
@@ -15,7 +15,7 @@ import (
 
 	"github.com/NebulousLabs/Sia/encoding"
 
-	"golang.org/x/crypto/blake2b"
+	"github.com/dchest/blake2b"
 )
 
 const (
